pandas/_libs/tslibs/parsing.pyi

```diff
@@ -27,11 +27,6 @@ def try_parse_dates(
     dayfirst: bool = ...,
     default: datetime | None = ...,
 ) -> npt.NDArray[np.object_]: ...
-def try_parse_year_month_day(
-    years: npt.NDArray[np.object_],  # object[:]
-    months: npt.NDArray[np.object_],  # object[:]
-    days: npt.NDArray[np.object_],  # object[:]
-) -> npt.NDArray[np.object_]: ...
```

Comment:
removing this function altogether, special-casing `%Y%m%d` just introduces bugs

The ISO8601 fastpath can handle this format just fine, let's use that

Reply:
does this impact perf?  im assuming this was introduced as a fastpath